Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -224,38 +254,48 @@ namespace O3DE::ProjectManager
             tr("The unique name for your gem consisting of only alphanumeric characters, '-' and '_'."),
             tr("A gem system name is required."));
         m_gemName->lineEdit()->setValidator(new QRegularExpressionValidator(QRegularExpression("[a-zA-Z]+[a-zA-Z0-9\\-\\_]*"), this));
+        m_gemName->setObjectName("createAGem");
```

Comment:
During the testing, I suspect that the rule for `createAGem` was not being applied properly, since there was improper indentation for the detail and creator detail pages. For some reason the selector did not enforce that for these line items unless I overrode the object name as you point out.

But yea, I am setting it for every `FormLineEditWidget`, so if there's a way to set it as a style for all instances of that widget for these two pages, that would be great to know.

Reply:
Hmm, yeah qss selectors can be tricky.  You may also want to try a selector that is more specific like this:
```
#createAGemRHS #formFrame {
}
```
I recommend trying more specific parent selectors to avoid adding all those object names

Reply:
would it be possible to do something like this?:
```
#createAGemRHS FormLineEditWidget {
...
}
```

Reply:
I would try it - qss changes will hot reload when you're running the project manager - so just edit the file and save it and wait to see what happens :)  - I sometimes will set something very visible like the background color in a selector to see if it is reaching the widgets I want to target